default/python/AI/colonization/calculate_production.py

```diff
@@ -142,37 +141,33 @@ def _get_production_bonus_unmodified(planet: fo.planet, stability: float):
             0.0,
             get_named_real("HONEYCOMB_TARGET_INDUSTRY_PERPOP"),
         ),
+        # TBD: Collective Thought Network? Ignored for now, AI doesn't know how to handle it.
+        # It shouldn't make a big difference for selecting which planets to colonise anyway.
     ]
-    for bonus in bonuses:
-        if bonus.available and stability >= bonus.min_stability:
```

Comment:
@Grummel7 
`bonus available ` is no longer taken into account.

I found it during small refactoring.
https://github.com/freeorion/freeorion/pull/3942